aten/src/ATen/templates/Tensor.h

```diff
@@ -288,7 +288,8 @@ class CAFFE2_API Tensor {
   T * data_ptr() const;
 
   template<typename T>
-  T * data() const {
+  C10_DEPRECATED_MESSAGE("tensor.data_ptr<T>() is deprecated. Please use tensor.data_ptr<T>() instead.")
+  T * data_deprecated() const {
```

Comment:
don't you need to leave this as `data()` for existing calls to work (and get the deprecated message?)

Reply:
Yes I was still trying to find all call sites of `data()` (and differentiate them from Caffe2 Tensor `.data()` and Storage `.data()`. I will rename this back to `data()` after finding all call sites of it.